### ISSUE TITLE
man page explanation of `--depth` option is confusing / wrong?

### ISSUE THREAD

**Issue by fredfisch** [ledger/ledger#1952]
The recently revised explanation of the `--depth` option is confusing and seems wrong to me:

> Limit the depth of the account tree.  In a balance report, for example,
> `--depth 2` will print balances only for accounts with two levels, i.e.
>    `Expenses:Entertainment`
> but not
>    `Expenses:Entertainment:Dining` .
> This is a display predicate, which means it only affects display,
> not the total calculations.  In register reports, `--depth` will collapse
> postings of child accounts beyond the given level and add
> their balances to the parent account.
 [Reference](https://github.com/ledger/ledger/blob/41ca4a7551cdc0e06496006d2adb1f62ba5dab5b/doc/ledger.1#L508)

To me this means, that `--depth` behaves differently for balance and register reports, respectively. Is this actually true? 

I'm currently on ledger-cli version 3.2.1 and in my tests the exact behaviour that has been recently documented in the current master branch for register reports is actually present for the balance report, too -- and to me this is exactly the way I want ledger to behave. And this behaviour should be reflected in the documentation.

Am I mistaken? Or did you intentionally make ledger behave differently in register and balance reports regarding the `--depth` option in the current master branch?

**Comment by tbm** [comment]
Adding @Rahix to clarify.

But I agree that `bal` and `reg` need the same behaviour.

**Comment by Rahix** [comment]
Yeah, the behavior is now the same for balance and register, that's what my PR was about.  The documentation could be rewritten to better clarify that, though, I agree ...

**Comment by tbm** [comment]
@Rahix can you submit a patch please.
